tests/phpunit/elementor/modules/atomic-widgets/prop-types/test-classes-prop-type.php

```diff
@@ -55,12 +55,12 @@ public function test_validate() {
 
 	public function invalid_classes_data_provider() {
 		return [
-			[ [ 'a', 'b', 3 ] ],
-			[ [ 'a', 'b', '3' ] ],
-			[ [ 'a', '2-b' ] ],
-			[ [ 'a', '-b' ] ],
-			[ [ 'a', '_b' ] ],
-			[ [ 'a', 'אבג' ] ],
+			[ [ [ '$$type' => 'string', 'value' => 'a'], [ '$$type' => 'string', 'value' => 'b'], [ '$$type' => 'number', 'value' => 3 ]] ],
+			[ [ [ '$$type' => 'string', 'value' => 'a'], [ '$$type' => 'string', 'value' => 'b'], [ '$$type' => 'string', 'value' => '3'] ] ],
+			[ [ [ '$$type' => 'string', 'value' => 'a'], [ '$$type' => 'string', 'value' => '2-b'] ] ],
+			[ [ [ '$$type' => 'string', 'value' => 'a'], [ '$$type' => 'string', 'value' => '-b' ]] ],
+			[ [ [ '$$type' => 'string', 'value' => 'a'], [ '$$type' => 'string', 'value' => '_b' ]] ],
+			[ [ [ '$$type' => 'string', 'value' => 'a'], [ '$$type' => 'string', 'value' => 'אבג'] ] ],
```

Comment:
Maybe like this, so it would be a bit more spacious, line-wrapped, and consistent 🙂

```suggestion
			[ [
				[ '$$type' => 'string', 'value' => 'a'],
				[ '$$type' => 'string', 'value' => 'b'],
				[ '$$type' => 'number', 'value' => 3 ]
			] ],
			[ [
				[ '$$type' => 'string', 'value' => 'a'],
				[ '$$type' => 'string', 'value' => 'b'],
				[ '$$type' => 'string', 'value' => '3']
			] ],
			[ [
				[ '$$type' => 'string', 'value' => 'a'],
				[ '$$type' => 'string', 'value' => '2-b']
			] ],
			[ [
				[ '$$type' => 'string', 'value' => 'a'],
				[ '$$type' => 'string', 'value' => '-b' ]
			] ],
			[ [
				[ '$$type' => 'string', 'value' => 'a'],
				[ '$$type' => 'string', 'value' => '_b' ]
			] ],
			[ [
				[ '$$type' => 'string', 'value' => 'a'],
				[ '$$type' => 'string', 'value' => 'אבג']
			] ],
```